sros2/sros2/policy/__init__.py

```diff
@@ -16,45 +16,48 @@
 
 from lxml import etree
 
-import pkg_resources
+try:
+    import importlib.resources as importlib_resources
+except ModuleNotFoundError:
+    # (clalancette): This fallback is necessary when importlib-resources is installed from pip.
+    # Note that we have to ignore the type, otherwise mypy throws a warning.
+    # See https://github.com/python/mypy/issues/1153 for details.
+    import importlib_resources  # type: ignore
 
 POLICY_VERSION = '0.2.0'
 
 
 def get_policy_default(name):
-    return pkg_resources.resource_filename(
-        package_or_requirement='sros2',
-        resource_name=os.path.join('policy', 'defaults', name))
+    with importlib_resources.path('sros2.policy.defaults', name) as path:
+        return str(path)
 
 
 def get_policy_schema(name):
-    return pkg_resources.resource_filename(
-        package_or_requirement='sros2',
-        resource_name=os.path.join('policy', 'schemas', name))
+    with importlib_resources.path('sros2.policy.schemas', name) as path:
+        return str(path)
 
 
 def get_policy_template(name):
-    return pkg_resources.resource_filename(
-        package_or_requirement='sros2',
-        resource_name=os.path.join('policy', 'templates', name))
+    with importlib_resources.path('sros2.policy.templates', name) as path:
+        return str(path)
 
 
 def get_transport_default(transport, name):
-    return pkg_resources.resource_filename(
-        package_or_requirement='sros2',
-        resource_name=os.path.join('policy', 'defaults', transport, name))
+    module = 'sros2.policy.defaults.' + transport
```

Comment:
Nit: we're moving towards using f-strings for all string formatting and concatenation in sros2, not a requirement for merging though, (same below)